Lilu/Sources/kern_devinfo.cpp

```diff
@@ -206,9 +206,17 @@ void DeviceInfo::grabDevicesFromPciRoot(IORegistryEntry *pciRoot) {
 				continue;
 
 			if (vendor == WIOKit::VendorID::Intel && (code == WIOKit::ClassCode::DisplayController || code == WIOKit::ClassCode::VGAController)) {
-				DBGLOG("dev", "found IGPU device %s", safeString(name));
-				videoBuiltin = obj;
-				requestedExternalSwitchOff |= videoBuiltin->getProperty(RequestedExternalSwitchOffName) != nullptr;
+					DBGLOG("dev", "found IGPU device %s", safeString(name));
```

Comment:
Why did you change indentation for this code?

Reply:
accidental change i believe